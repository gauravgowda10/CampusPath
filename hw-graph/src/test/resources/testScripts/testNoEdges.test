# Tests adding two nodes

# Create a graph
CreateGraph twoNodesGraph

# Add two nodes
AddNode twoNodesGraph x
AddNode twoNodesGraph y

# Print the nodes and edges in the graph
ListNodes twoNodesGraph
ListChildren twoNodesGraph x
ListChildren twoNodesGraph y
