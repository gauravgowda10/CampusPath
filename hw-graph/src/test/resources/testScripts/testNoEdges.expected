# Tests adding two nodes

# Create a graph
created graph twoNodesGraph

# Add two nodes
added node x to twoNodesGraph
added node y to twoNodesGraph

# Print the nodes and edges in the graph
twoNodesGraph contains: x y
the children of x in twoNodesGraph are:
the children of y in twoNodesGraph are:
